report-pox-krypton.js

```diff
@@ -688,7 +688,15 @@ function process_burnchain_ops() {
 
     console.log(block.block_height,
       // block.block_commits.map(bc => `${bc.leader_key_address.substring(0, 10)}${bc.txid === block.winning_block_txid ? '*' : ' '}(${bc.key_block_ptr})`).sort((a, b) => a.localeCompare(b)).join(' '),
-      block.block_commits.map(bc => `${bc.leader_key_address.substring(0, 10)}${bc.txid === block.winning_block_txid ? '*' : ' '}`).sort((a, b) => a.localeCompare(b)).join(''),
+      block.block_commits
+        .map(
+          (bc) =>
+            `${bc.leader_key_address.substring(0, 10)} ${bc.burn_fee}${
+              bc.txid === block.winning_block_txid ? "*" : " "
+            }`
+        )
+        .sort((a, b) => a.localeCompare(b))
+        .join(""),
```

Comment:
can you not change the code formatting, makes it harder to see the real change, keep single quotes when they are used, and make the display of the `burn_fee` optional (using a new command line option).  So far the display even with 20 miners stays on one line on a large screen, but this would make it go over, and harder to read.